web/src/notifications.js

```diff
@@ -728,7 +759,7 @@ export function register_click_handlers() {
             const message_id = $(e.currentTarget).data("message-id");
             message_lists.current.select_id(message_id);
             navigate.scroll_to_selected();
-            compose_banner.clear_message_sent_banners();
+            compose_banner.clear_message_sent_banners(false);
```

Comment:
Hmm, why `false` here? I would expect if you're clicking a button on a banner, you've had the chance to look at both of them, and are probably happier to just have the space cleared.

Reply:
I can see why a user might like that, but while testing I remember thinking it seemed buggy when the unmute banner disappeared on clicking the scroll to message link in another banner, after which I added the `false` here.

@alya Would like to hear what you think about this - if on sending a message to a muted topic / stream, the unmute banner shows up along with the "Scroll to the message you just sent" banner, and if the user 1st clicks the link to scroll to the message, should we also clear the unmute banner or leave it undisturbed?

Reply:
Yeah, that's fair; I'm happy to defer to your judgement having actually experienced this detail in testing and feeling like ti was buggy.